src/order/filter/basic.lean

```diff
@@ -1104,6 +1107,12 @@ lemma tendsto_cong {f₁ f₂ : α → β} {l₁ : filter α} {l₂ : filter β}
   (h : tendsto f₁ l₁ l₂) (hl : {x | f₁ x = f₂ x} ∈ l₁.sets) : tendsto f₂ l₁ l₂ :=
 by rwa [tendsto, ←map_cong hl]
 
+theorem tendsto.congr {f₁ f₂ : α → β} {l₁ : filter α} {l₂ : filter β}
+    (h : tendsto f₁ l₁ l₂) (h' : ∀ x, f₁ x = f₂ x) :
+  tendsto f₂ l₁ l₂ :=
```

Comment:
this should be an iff

Reply:
Actually, is this necessary at all? This is just `congr; ext`

Reply:
So the proofs that use this theorem should use `convert` instead?

Reply:
Oh, I see now how this can come up in a proof. I guess this version also has its uses, although the arguments should be flipped (it will still work with projection notation). But I think this can be one line proved by `by rw <- funext h'; exact h` or `by convert h; ext; apply h'`